test/save_load_scfres.jl

```diff
@@ -1,27 +1,27 @@
-# This test is based on example examples/collinear_magnetism.jl
+# This test is based on example docs/src/guide/tutorial.jl
 # Import statements are not all included in front to test for 
 # error statements in dispatch
 
 using DFTK
+using Unitful
+using UnitfulAtomic
 
-a = 5.42352
-lattice = a / 2 * [[-1  1  1];
-                   [ 1 -1  1];
-                   [ 1  1 -1]]
-Fe = ElementPsp(:Fe, psp=load_psp("hgh/lda/Fe-q8.hgh"))
-atoms = [Fe => [zeros(3)]];
-kgrid = [3, 3, 3]
-Ecut = 15
-magnetic_moments = [Fe => [4, ]];
-model = model_LDA(lattice, atoms, magnetic_moments=magnetic_moments, temperature=0.01)
+a = 5.431u"angstrom"
+lattice = a / 2 * [[0 1 1.];
+                   [1 0 1.];
+                   [1 1 0.]]
+Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
+atoms = [Si => [ones(3)/8, -ones(3)/8]]
```

Comment:
Actually in tests we tend to use the `testcases.jl` file, which stores some of this data once and for all ... see the other tests for how to do that.